test/OpenTelemetry.Instrumentation.ConfluentKafka.Tests/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Create a container for running the OpenTelemetry Redis integration tests.
+# This should be run from the root of the repo:
+# docker build --file test/OpenTelemetry.Instrumentation.ConfluentKafka.Tests/Dockerfile .
+
+ARG BUILD_SDK_VERSION=8.0
+ARG TEST_SDK_VERSION=8.0
+
+FROM mcr.microsoft.com/dotnet/sdk:${BUILD_SDK_VERSION} AS build
+ARG PUBLISH_CONFIGURATION=Release
+ARG PUBLISH_FRAMEWORK=net8.0
+WORKDIR /repo
+COPY . ./
+WORKDIR "/repo/test/OpenTelemetry.Instrumentation.ConfluentKafka.Tests"
+RUN dotnet publish "OpenTelemetry.Instrumentation.ConfluentKafka.Tests.csproj" -c "${PUBLISH_CONFIGURATION}" -f "${PUBLISH_FRAMEWORK}" -o /drop -p:IntegrationBuild=true -p:TARGET_FRAMEWORK=${PUBLISH_FRAMEWORK}
```

Comment:
What does `/p:IntegrationBuild=true` do?

Reply:
It avoids taking a private dependency on `MinVer`. See https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/e42f6cdda1120e7630aa69e0ab7e8560feddba31/build/Common.prod.props#L23